hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -560,6 +560,67 @@
     }
   ]
 
+  features: [
+    {
+      name: "OTP_CTRL.VENDOR_TEST_PARTITION"
```

Comment:
I suggest OTP_CTRL.PARTITION.VENDOR_TEST for this ans others. Follow the same pattern as OTP_CTRL.PARTITION.SECRET1 below, for example. This is also consistent with https://github.com/lowRISC/opentitan/issues/19444.

Reply:
Updated